src/functions/sendslack/handler.ts

```diff
@@ -0,0 +1,130 @@
+import type { ValidatedEventAPIGatewayProxyEvent } from "@libs/api-gateway";
+import { formatJSONResponse } from "@libs/api-gateway";
+import { middyfy } from "@libs/lambda";
+require("dotenv").config({ path: __dirname + "/../../../../../.env" });
+import { WebClient, LogLevel } from "@slack/web-api";
+import moment from 'moment';
```

Comment:
Use double quotes with strings
```suggestion
import moment from "moment";
```